distribution/docs/src/main/resources/content/_reference/_dependencyList/ddf-dependency-list.adoc

```diff
@@ -3,361 +3,9 @@
-* org.jvnet.ogc:gml-v_3_2_1-schema:jar:1.1.0
-* org.jvnet.ogc:gml-v_3_2_1:pom:1.1.0
-* org.jvnet.ogc:ogc-tools-gml-jts:jar:1.0.3
-* org.jvnet.ogc:ows-v_1_0_0-schema:jar:1.1.0
-* org.jvnet.ogc:ows-v_1_0_0:jar:2.6.1
-* org.jvnet.ogc:ows-v_1_1_0-schema:jar:1.1.0
-* org.jvnet.ogc:ows-v_2_0:jar:2.6.1
-* org.jvnet.ogc:wcs-v_1_0_0-schema:jar:1.1.0
-* org.jvnet.ogc:wfs-v_1_1_0:jar:2.6.1
-* org.jvnet.ogc:wps-v_2_0:jar:2.6.1
-* org.la4j:la4j:jar:0.6.0
-* org.locationtech.jts:jts-core:jar:1.15.0
-* org.locationtech.spatial4j:spatial4j:jar:0.6
-* org.locationtech.spatial4j:spatial4j:jar:0.7
-* org.mockito:mockito-core:jar:1.10.19
-* org.noggit:noggit:jar:0.6
-* org.objenesis:objenesis:jar:2.5.1
-* org.objenesis:objenesis:jar:2.6
-* org.openexi:nagasena-rta:jar:0000.0002.0049.0
-* org.openexi:nagasena:jar:0000.0002.0049.0
-* org.opensaml:opensaml-core:jar:3.3.0
-* org.opensaml:opensaml-soap-impl:jar:3.3.0
-* org.opensaml:opensaml-xmlsec-api:jar:3.3.0
-* org.opensaml:opensaml-xmlsec-impl:jar:3.3.0
-* org.ops4j.pax.exam:pax-exam-container-karaf:jar:4.11.0
-* org.ops4j.pax.exam:pax-exam-junit4:jar:4.11.0
-* org.ops4j.pax.exam:pax-exam-link-mvn:jar:4.11.0
-* org.ops4j.pax.exam:pax-exam:jar:4.11.0
-* org.ops4j.pax.swissbox:pax-swissbox-extender:jar:1.8.2
-* org.ops4j.pax.tinybundles:tinybundles:jar:2.1.1
-* org.ops4j.pax.url:pax-url-aether:jar:2.4.5
-* org.ops4j.pax.url:pax-url-wrap:jar:2.4.5
-* org.ops4j.pax.web:pax-web-api:jar:6.0.9
-* org.osgi:org.osgi.compendium:jar:4.3.1
-* org.osgi:org.osgi.compendium:jar:5.0.0
-* org.osgi:org.osgi.core:jar:4.3.1
-* org.osgi:org.osgi.core:jar:5.0.0
-* org.osgi:org.osgi.enterprise:jar:5.0.0
-* org.ow2.asm:asm:jar:5.0.2
-* org.ow2.asm:asm:jar:5.0.4
-* org.parboiled:parboiled-core:jar:1.1.8
-* org.parboiled:parboiled-java:jar:1.1.8
-* org.quartz-scheduler:quartz-jobs:jar:2.2.3
-* org.quartz-scheduler:quartz:jar:2.1.7
-* org.quartz-scheduler:quartz:jar:2.2.3
-* org.rrd4j:rrd4j:jar:2.2
-* org.rrd4j:rrd4j:jar:3.2
-* org.simplejavamail:simple-java-mail:jar:4.1.3
-* org.slf4j:jcl-over-slf4j:jar:1.7.24
-* org.slf4j:jul-to-slf4j:jar:1.7.24
-* org.slf4j:slf4j-api:jar:1.7.12
-* org.slf4j:slf4j-api:jar:1.7.1
-* org.slf4j:slf4j-api:jar:1.7.24
-* org.slf4j:slf4j-ext:jar:1.7.1
-* org.slf4j:slf4j-log4j12:jar:1.7.12
-* org.slf4j:slf4j-log4j12:jar:1.7.24
-* org.slf4j:slf4j-log4j12:jar:1.7.7
-* org.slf4j:slf4j-simple:jar:1.7.1
-* org.slf4j:slf4j-simple:jar:1.7.5
-* org.spockframework:spock-core:jar:1.1-groovy-2.4
-* org.springframework.ldap:spring-ldap-core:jar:2.3.2.RELEASE
-* org.springframework.osgi:spring-osgi-core:jar:1.2.1
-* org.springframework:spring-core:jar:5.0.4.RELEASE
-* org.taktik:mpegts-streamer:jar:0.1.0_2
-* org.twitter4j:twitter4j-core:jar:4.0.4
-* org.xmlunit:xmlunit-matchers:jar:2.5.1
-* us.bpsm:edn-java:jar:0.4.4
-* xalan:serializer:jar:2.7.2
-* xalan:xalan:jar:2.7.2
-* xerces:xercesImpl:jar:2.11.0
-* xerces:xercesImpl:jar:2.9.1
-* xml-apis:xml-apis:jar:1.4.01
-* xpp3:xpp3:jar:1.1.4c
+include::maven-dependency-list.txt[]
+
```

Comment:
we may want to have a some sort of header or separator to distinguish the two lists as they are presented alphabetically but it arbitrarily restarts about halfway through.

Maybe drop the period from the end of the `:summary:` property and adding ` - Java` and ` - JavaScript` to the end of the `.{summary} lines.